### PR TITLE
Ruby Version 2.7

### PR DESCRIPTION
**Pull request by gkostin1966**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 SHELL = /bin/bash
 
 build_tag ?= dul-arclight
-builder_image ?= gitlab-registry.oit.duke.edu/devops/containers/ruby:2.6-main
+builder_image ?= gitlab-registry.oit.duke.edu/devops/containers/ruby:2.7-main
```
Comment (estelendur) [inline Makefile:4]: actually I think we can just delete this Makefile. I can take on the small task of "rip out Duke's build stuff we aren't using any more and update the README(s) accordingly"

**Review by botimer — APPROVED** [review]
Looks good. I didn't audit the gem updates, but they look pretty conservative -- nice to see them get some attention.

**Review by respinos — APPROVED** [review]
🤞

File: Makefile (hunk 1 above)
Comment (gkostin1966) [reply Makefile:4]: Sounds good to me.

**Pull request merged**